src/components/MAPI/organizations/__tests__/OrganizationList.tsx

```diff
@@ -0,0 +1,323 @@
+      .reply(StatusCodes.Ok, randomClusterList);
+
+    render(getComponent({}));
+
+    expect(
+      screen.getByRole('button', { name: 'Add organization' })
+    ).toBeInTheDocument();
+  });
+
+  it('can create an organization', async () => {
+    const randomClusterList = generateClusterList([]);
+
+    nock(window.config.mapiEndpoint)
+      .post('/apis/security.giantswarm.io/v1alpha1/organizations/')
+      .reply(StatusCodes.Ok, {
+        apiVersion: 'security.giantswarm.io/v1alpha1',
+        kind: 'Organization',
+        metadata: {
+          name: 'some-org',
+        },
+        spec: {},
+        status: {
+          namespace: 'org-some-org',
+        },
+      });
+
+    nock(window.config.mapiEndpoint)
+      .post('/apis/authorization.k8s.io/v1/selfsubjectaccessreviews/')
+      .reply(
+        StatusCodes.Ok,
+        authorizationv1Mocks.selfSubjectAccessReviewCanListOrgs
+      );
+    nock(window.config.mapiEndpoint)
+      .get('/apis/security.giantswarm.io/v1alpha1/organizations/')
+      .reply(StatusCodes.Ok, securityv1alpha1Mocks.getOrganizationListResponse);
+
+    nock(window.config.mapiEndpoint)
+      .get('/apis/cluster.x-k8s.io/v1alpha3/clusters/')
+      .reply(StatusCodes.Ok, randomClusterList);
+
+    render(getComponent({}));
+
+    const addOrgnizationButton = screen.getByRole('button', {
+      name: 'Add organization',
+    });
+    fireEvent.click(addOrgnizationButton);
+
+    const createOrganizationButton = screen.getByRole('button', {
+      name: 'Create organization',
+    });
+    expect(createOrganizationButton).toBeDisabled();
+
+    const nameInput = screen.getByLabelText('Name');
+    fireEvent.change(nameInput, {
+      target: { value: 'some-org' },
+    });
+
+    expect(createOrganizationButton).not.toBeDisabled();
+
+    fireEvent.click(createOrganizationButton);
+
+    expect(await screen.findByText('some-org')).toBeInTheDocument();
+    expect(
+      await withMarkup(screen.findByText)(
+        'Organization some-org created successfully'
+      )
+    );
+    expect(
+      screen.queryByRole('button', {
+        name: 'Create organization',
+      })
+    ).not.toBeInTheDocument();
+  });
+
+  it('can cancel an organization creation', async () => {
+    const randomClusterList = generateClusterList([]);
+
+    nock(window.config.mapiEndpoint)
+      .get('/apis/cluster.x-k8s.io/v1alpha3/clusters/')
+      .reply(StatusCodes.Ok, randomClusterList);
+
+    render(getComponent({}));
+
+    const addOrgnizationButton = screen.getByRole('button', {
+      name: 'Add organization',
+    });
+    fireEvent.click(addOrgnizationButton);
+
+    const nameInput = screen.getByLabelText('Name');
+    fireEvent.change(nameInput, {
+      target: { value: 'not-created-org' },
+    });
+
+    const cancelButton = screen.getByRole('button', {
+      name: 'Cancel',
+    });
+    fireEvent.click(cancelButton);
+
+    expect(screen.queryByText('not-created-org')).not.toBeInTheDocument();
```

Comment:
Is this to check if the success message is not present?

Reply:
I was thinking to check that the user input organization name is not present anywhere (both the input field and the success message). Should this be two separate checks actually?

Reply:
For the input field value there should be an extra one (i.e. `screen.queryByDisplayValue()`)